support/globalconfig/network.go

```diff
@@ -25,7 +28,17 @@ func ReconcileNetworkConfig(cfg *configv1.Network, hcp *hyperv1.HostedControlPla
 	for _, entry := range hcp.Spec.Networking.ClusterNetwork {
 		hostPrefix := uint32(entry.HostPrefix)
 		if hostPrefix == 0 {
-			hostPrefix = defaultHostPrefix
+			ipv4, err := util.IsIPv4(entry.CIDR.String())
+			if err != nil {
+				fmt.Printf("error checking the ClusterNetwork CIDR %s: %v\n", entry.CIDR.String(), err)
```

Comment:
Why is this error being ignored?

Is this a req for valid input it should be validated here in `validateNetworks()` as part of this PR. Only then it would be safe to ignore here. And even then it'd still make sense to let this function or this particular piece of code to return err

Additionally If we need to print anything, it needs to go through Log. An even then it'd be

Reply:
At this point, the Cluster and Service networks should fail early on that `validateNetworks()` function when it is parsed by the hosterClusterController. The function `ReconcileNetworkConfig` is basically consumed by HCP and ignoring the error here should not create side-scenarios for failures. 

Regarding the print msg I can drop, it was useful for initial debug.